api/versions_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/cadvisor/events"
+)
+
+func TestGetEventRequestBasicRequest(t *testing.T) {
```

Comment:
consider using "github.com/stretchr/testify/assert" This is used by the integration tests and it makes a lot of thing much easier :)
e.g.:
```Go
assert.False(t, shouldGetAllEvents)
assert.NotNil(t, err)
```